scripts/mkcloud

```diff
@@ -638,6 +620,39 @@ function macfunc()
     printf "52:54:77:77:77:%02x" $1
 }
 
+function get_admin_node_dist()
+{
+    # echo the name of the current dist for the admin node
+    local dist=
+    case $(getcloudver) in
+        7)  dist=SLE12SP1
+            [[ $want_sles12sp2 ]] && dist=SLE12SP2
+            ;;
+        6)  dist=SLE12SP1 ;;
+        5)  dist=SLE11
+            [[ $want_sles12 ]] && dist=SLE12
```

Comment:
I guess that was meant for the lonelynode codepath which now goes through onhost_deploy_image -> get_admin_node_image_name -> get_admin_node_dist.
but it seem to be wrong as is.